ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Install pmm-dump, always overwrite the version before with latest
+
+- name: Download pmm-dump binary (tech preview)
+  get_url:
+    url: https://github.com/percona/pmm-dump/releases/download/0.5.1-beta/pmm-transferer
+    dest: /tmp/pmm-transferer
+    tmp_dest: /tmp/
+    force: yes
+    headers:
+      Accept: "application/octet-stream"
```

Comment:
It won't work without internet :(

Reply:
Should we just put the file at build time, not download it?

Reply:
I removed the playbook in favor of a separate spec.